platform-sdk/swirlds-platform-core/src/test/resources/com/swirlds/platform/crypto/EnhancedKeyStoreLoader/enhanced-invalid-case-1/t-private-alice.tss

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
+AVS7ccxMt5screfvBWYJkUhm3SRhdkNxvsk8KcDMEXgl
+-----END PRIVATE KEY-----
```

Comment:
Would add newlines for consistency, etc.

Reply:
Fixed all test keys.